tests/translator/input/error_function_with_event_dest_type.yaml

```diff
@@ -2,16 +2,6 @@ Parameters:
   SNSArn:
     Type: String
     Default: my-sns-arn
-Globals:
```

Comment:
note: move the property values from global to resource level so that we can test multiple errors in the same template